### PR TITLE
Replicate fit decode example

### PR DESCRIPTION
**Pull request by quamen**
This pull request replicates the decode example from the FIT SDK.
## TLDR
- Loads the provided fit.jar library file.
- Sets up minitest as the test framework.
- Replicates the interface of the existing FIT and TCX gems to read an IO stream and parse the FIT file.
- Tests that we can parse one of my Forerunner 620 FIT files.
## One more time with details:

This means we can now read out the following information from the `file_id` message:
- type
- manufacturer
- product
- serial_number

There are no smarts around what the values mean, but we are checking for invalid values before assigning them to the `file_id` object. The numbers that we get back are the same ones we get back if we run the `FitDecodeFit.jar` example program from the command line:

```
☁  forerunner [replicate-fit-decode-example] java -jar FitDecodeExample.jar 620.FIT
FIT Decode Example Application
File ID:
   Type: 4
   Manufacturer: 1
   Product: 1623
   Serial Number: 3877468399
Decoded FIT file 620.FIT.
```

### CODE REVIEW

**Comment by quamen** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/531290cfe30ba021d600edbf/compare/replicate-fit-decode-example">analyzed this pull request</a>.

**Comment by plasticine** [comment]
Pretty cool! Why mini-test?

**Comment by quamen** [comment]
minitest: because it's built into ruby and it means one less dependency.

Thought I would give it a crack after watching the talk on it at rubyconf.

**Comment by plasticine** [comment]
Sure, just curious. I’ve never used it. :P

**Comment by quamen** [comment]
Neither have I. But I figure since it's bundled into ruby it should get some love. 

Anyway, have a play with the code and see if you can get the repo up and running. If you can then I'll merge it and get to work on the other fields we currently use.
